src/librustc/ty/steal.rs

```diff
@@ -43,4 +43,8 @@ impl<T> Steal<T> {
         let value = value_ref.take();
         value.expect("attempt to read from stolen value")
     }
+
+    pub fn stolen(&self) -> bool {
```

Comment:
Is this used anywhere? I don't see it anywhere else in the diff.

Reply:
It was used in a previous version. I'll remove it.